### PR TITLE
Add reference to PAM RFC-86

### PR DESCRIPTION
**Pull request by wilder**

### CODE REVIEW

**Comment by MrPickles** [comment]
Thanks for the pull request!

So in this case the "PAM" module on the `README` actually refers to the [PAM Module that we have on Github](https://github.com/google/google-authenticator-libpam), not necessarily the RFC itself. As a result, the choice of link might be a bit confusing. However, there's a link to the GitHub project a few lines below, so I guess this link would be okay.

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ officially supported Google product.
 ## Description
 
 The Google Authenticator project includes implementations of one-time passcode
-generators for several mobile platforms, as well as a pluggable authentication
-module (PAM). One-time passcodes are generated using open standards developed by
+generators for several mobile platforms, as well as a Pluggable Authentication
+Module [(PAM)][PAM]. One-time passcodes are generated using open standards developed by
```
Comment (MrPickles) [inline README.md]: This line exceeds 80 characters. Do you mind enforcing the line lengths? It'll end up looking like this:

```
Module [(PAM)][PAM]. One-time passcodes are generated using open standards
developed by the [Initiative for Open Authentication (OATH)][OATH] (which is
unrelated to [OAuth][]).
```

**Comment by wilder** [comment]
Oh, it makes sense... I though it was referring to the RFC!

**Comment by wilder** [comment]
So it makes sense to leave it without the reference to any link, right? Or is better to also reference the PAM Module?

**Comment by MrPickles** [comment]
> So is it better to leave it without the reference to any link or to make it also refer to the PAM Module?

You can refer to the RFC. It's fine.

**Comment by wilder** [comment]
Thanks!

**Pull request merged**